azure-pipelines.yml

```diff
@@ -152,14 +157,23 @@ stages:
               includeNuGetOrg: true
               arguments: 'install --global CycloneDX'
 
-          - task: DotNetCoreCLI@2
+          - powershell: |
+              # It's not possible to use the tool using only its name (dotnet CycloneDX [args]), because of this message in the pipeline:
+              # "Since you just installed the .NET Core SDK, you will need to reopen the Command Prompt window before running the tool you installed."
+              # Instead, we need to find the installed tool and execute it directly using its absolute path.
```

Comment:
Why did it work before?

Reply:
That's a good question. This is what I've found:
https://github.com/dotnet/sdk/issues/8999
Based on that the difference must be the way the .NET SDK is installed on the new CI image. I didn't check that, because [the master build is still red there](https://github.com/SonarSource/dotnet-ci-images/runs/26230827721), so I can't really make modifications to the image. So I did a workaround to make the build for this PR green.